client/src/components/Invite/Invite.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useRef } from 'react'
+
+import './Invite.scss'
+import Button from '../Button'
+import useCooldown from '../../hooks/useCooldown'
+
+export interface InviteProps {
+  gameUrl: string
+}
+
+const Invite: FC<InviteProps> = ({ gameUrl }) => {
+  const [isCoolingDown, triggerCooldown] = useCooldown(1500)
+  const linkInputRef = useRef<HTMLInputElement>(null)
+  const buttonRef = useRef<HTMLButtonElement>(null)
+
+  const selectLinkText = () => {
+    linkInputRef.current && linkInputRef.current.select()
+  }
+
+  const copyToClipboard = () => {
+    selectLinkText()
+    document.execCommand('copy')
+    buttonRef.current && buttonRef.current.focus()
```

Comment:
I'm 99% sure we can even get there without the ref being initialized
```suggestion
    buttonRef.current!.focus()
```